### ISSUE TITLE
The radio button should be toggle only after clicking on the radio button's in admin panel

### ISSUE THREAD

**Issue by Sanjaybhattwebkul** [bagisto/bagisto#6390]
**Bagisto Version:** Master

**Steps To reproduce**--

**1.** Log in as an admin.
**2.** Click on the configuration tab.
**3.** Click outside of the **Compare Options** radio button.

**Expected result:** The radio button should be **toggle only** after clicking on the **radio button's** in admin panel

**Actual result:** The radio button is toggling while clicking on the outside of the radio buttons in the admin panel. Please find the screen recording [Here](https://www.awesomescreenshot.com/video/8731461?key=f5495117b78922a8df513dc0d876fd3f)

**Comment by Sanjaybhattwebkul** [comment]
@jitendra-webkul 
I am reopening this issue Because this **PR** will be reverted.

**Comment by Sanjaybhattwebkul** [comment]
@jitendra-webkul 
As we discuss we need to raise **a new Issue** for the same, so I am closing this issue.
